typescript/sdk/src/token/adapters/SealevelTokenAdapter.ts

```diff
@@ -283,11 +288,34 @@ export abstract class SealevelHypTokenAdapter
     return undefined;
   }
 
+  // The sender is required, as simulating a transaction on Sealevel requires
+  // a payer to be specified that has sufficient funds to cover the transaction fee.
   async quoteTransferRemoteGas(
-    _destination: Domain,
+    destination: Domain,
+    sender?: Address,
   ): Promise<InterchainGasQuote> {
-    // TODO Solana support
-    return { amount: 0n };
+    const tokenData = await this.getTokenAccountData();
+    const destinationGas = tokenData.destination_gas?.get(destination);
+    if (destinationGas === undefined) {
```

Comment:
consider `isNullish` or just `!`. Sorry to keep nitpicking on these but it's good for JS code to be defensive against more falsy values than just undefined